internal/common/os_name.bzl

```diff
@@ -66,3 +66,17 @@ def is_darwin_os(rctx):
 def is_linux_os(rctx):
     name = os_name(rctx)
     return name == OS_NAMES[3] or name == OS_NAMES[4] or name == OS_NAMES[5]
+
+def node_exists_for_os(node_version, os_name):
+    "Whether a node binary is available for this platform"
+    node_major_version = int(node_version.split(".")[0])
```

Comment:
Maybe we could reuse [check_version](https://github.com/bazelbuild/rules_nodejs/blob/stable/internal/common/check_version.bzl#L52) here?

Reply:
good point, done